tests/bugs/gh-463.slang.expected

```diff
@@ -1,6 +1,8 @@
 result code = -1
 standard error = {
 tests/bugs/gh-463.slang(7): error 30015: undefined identifier 'COUNT'.
+(0): note 31122: see defintion for attribute parameter 'instance.count'
+tests/bugs/gh-463.slang(7): error 30015: undefined identifier 'COUNT'.
```

Comment:
This error message seems like it is related to double-checking of attribute arguments, and that seems like a big problem to me. I don't think this is a reasonable change to the expected output for this test.

Reply:
Fixed.